program/find-average-of-numbers-by-recursion/FindAverageOfNumbersByRecursion.ts

```diff
@@ -0,0 +1,12 @@
+function average_of_numbers(arr: any, length: number, i: number): any {
+    if (i == length - 1) {
+      return arr[i];
```

Comment:
can we use strict equality operator for comparison ?

Reply:
I have changed to strict equality either way so as to be in line with Javascript programming style.

but typescript is already having type definitions binded to variables.
if we try to assign different type, it would be compilation error.